### PR TITLE
build: upgrade to Bazel 0.8

### PR DESCRIPTION
**Pull request by alexeagle**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
           <<: *post_checkout
       - restore_cache:
           key: angular-{{ .Branch }}-{{ checksum "yarn.lock" }}
-
+      - run: bazel info release
```
Comment (IgorMinar) [inline .circleci/config.yml]: is this just to log version info?

**Review by IgorMinar — APPROVED** [review]
This is looking good if CI passes.

Also please add Closes #20807 to the commit message.

File: .circleci/config.yml (hunk 1 above)
Comment (alexeagle) [reply .circleci/config.yml]: Yes

**Comment by mgol** [comment]
I submitted a followup PR as `.nvmrc` was not modified in this one. See #20831.

**Pull request closed without merging**